src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/ArticlePageObject.java

```diff
@@ -797,4 +801,8 @@ public void verifyURLVEActionEditEditor(Editor expectedEditor, String wikiURL) {
             "Invalid expected editor chosen: " + expectedEditor.name());
     }
   }
+
+  public boolean isFeaturedVideo() {
+    return featuredVideo.isDisplayed();
```

Comment:
`wait.forElementVisible()` should be used before

Reply:
done